src/main/webapp/web.xml

```diff
@@ -4,11 +4,101 @@
                       http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd"
          version="4.0">
     <servlet>
```

Comment:
i prefer to use annotations such as WebServlet or WebFilter, but it's up to you)

Reply:
i also agree that using `@WebServlet` is very convenient
but you can leave as it is, it's fine